app/public/js/controllers/activity.js

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
The activity-list controller manages the creation and deletion. The activity manages the modification and realtime start and stop.